docs/dev/release.rst

```diff
@@ -188,20 +183,78 @@ You can create a clean checkout as follows (requires
      Other Changes and Additions
      ---------------------------
 
-#. Commit your changes of the, uh, change log::
+     A.C.1 (unreleased)
+     ==================
 
-     git add CHANGES.rst
-     git commit -m "Back to development: A.B.dev"
+     Bug Fixes
+     ---------
 
-#. Push out the updated code and tag. If applicable, change ``origin`` to point to
-   the remote that points to the repository being released::
+#. Commit your changes of the, uh, change log with a message, "Back to development: A.B.dev"
 
-     git push origin main
-     git push origin vX.Y.Z
+#. Follow procedures for :ref:`release-milestones`.
+
+#. For your own sanity unrelated to the release, grab the new tag for your fork::
+
+     git fetch upstream --tags
+
+Congratulations, you have just released a new version of Jdaviz!
+
+.. _release-bugfix:
+
+Releasing a bugfix version
+==========================
+
+The procedure for a bugfix release is a little different from a feature release - you will
+be cherry-picking bugfixes into an existing release branch, and will also need to do some
+cleanup on the ``main`` branch.
+
+#. Lock down the ``main`` branch of the repository by setting the
+   `branch protection <https://github.com/spacetelescope/jdaviz/settings/branches>`_
+   rule for ``main`` to some high number required to merge, so that more PRs don't
+   get merged while you're releasing.
+
+#. Review the ``CHANGES.rst`` file on ``main`` to see which PRs were listed in the unreleased
+   bugfix version section (this should be immediately below the unreleased feature section).
+   For example, if the top section is for 3.1, there should be an unreleased 3.0.X section
+   below that, where X is a number greater than 0.
```

Comment:
maybe better to suggest filtering closed PRs with the milestone in case any have "no-changelog-needed" (realized this while milestoning https://github.com/spacetelescope/jdaviz/pull/1752 for backporting but it doesn't need a changelog).

Reply:
Maybe we can modify the stuff in https://github.com/astropy/astropy-tools/tree/main/pr_consistency to help sanity check? Might be tedious to have to do this manually every bug fix release.